src/main/java/training/Items.java

```diff
@@ -0,0 +1,27 @@
+package training;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+
+import com.amazonaws.services.dynamodbv2.document.DynamoDB;
+import com.amazonaws.services.dynamodbv2.document.Item;
+import com.amazonaws.services.dynamodbv2.document.PutItemOutcome;
+import com.amazonaws.services.dynamodbv2.document.Table;
+
+public class Items {
+    public void putItem() {
+
+        AmazonDynamoDB client = AmazonDynamoDBClientBuilder.standard().build();
+        DynamoDB dynamoDB = new DynamoDB(client);
+
+        Table table = dynamoDB.getTable("IGHItems");
+
+        Item item = new Item()
+                .withPrimaryKey("ghid", 1)
+                .withString("Title", "cycles")
+                .withNumber("shelflife", 12365)
+                .withBoolean("discontinued", false);
+
+        table.putItem(item);

```

Comment:
You did not use the outcome, you should use it so that you can find out if item is inserted or not.